TUTORIAL.md

```diff
@@ -357,7 +357,7 @@ If you have any problems feel free to [raise an issue](https://github.com/wrld3d
   Troubleshooting
 ===================
 #### <a name="multipolygons"/>Multipolygons
-If you are receiving an error related to multipolygons in your submission, first check that you ran the [Multipart to Singleparts](#export-level-to-geojson) conversion step when you exported your layer to geojson.
+If you are receiving an error related to multipolygons in your submission, first check that you ran the [Multipart to Singleparts](#export-level-to-geojson) conversion step when you exported your layer to GeoJSON.
```

Comment:
As per comment in README.md this section can be factored out and replaced by a link to the general troubleshooting section.